src/systems/vehicle_damage/VehicleDamage.hh

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef IGNITION_GAZEBO_SYSTEMS_VEHICLE_DAMAGE_HH_
+#define IGNITION_GAZEBO_SYSTEMS_VEHICLE_DAMAGE_HH_
+
+#include <memory>
+#include <ignition/gazebo/config.hh>
+#include <ignition/gazebo/Export.hh>
+#include <ignition/gazebo/System.hh>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declarations.
+  class VehicleDamagePrivate;
+
+  /// \brief A system that simulates a simple wind model.
+  /// The wind is described as a uniform worldwide model. So it is independent
+  /// from model position for simple computations. Its components are computed
+  /// separately:
```

Comment:
Could you update the docs and add some example usage?

Reply:
Partial fix in 7c84f8ed87f1f3997138d1859c9f95a3d63078e8